src/main/webapp/WEB-INF/view/profile.jsp

```diff
@@ -84,21 +89,22 @@ List<Message> messagesByUser = (List<Message>) request.getAttribute("messagesByU
       </form>
 
       <hr/>
-    <% } %>
 
     <h1><%=request.getSession().getAttribute("user")%>'s Sent Messages</h1>
```

Comment:
Lines 93-106 should be indented two more spaces since they are inside the if-statement that starts on line 76 and ends on line 107.